### ISSUE TITLE
[Bug]: Couldn't Sign You In

### ISSUE THREAD

**Issue by psaunders** [th-ch/youtube-music#2036]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.3.6

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official version of (WEB) YTM](https://music.youtube.com).

### What operating system are you using?

macOS

### Operating System Version

14.4.1

### What CPU architecture are you using?

arm64 (including Apple Silicon)

### Last Known Working YouTube Music (Application) version

3.3.6

### Reproduction steps

When I go to sign-in it does fetch my google account.
I click "Next"
<img width="223" alt="image" src="https://github.com/th-ch/youtube-music/assets/424956/8193469c-8505-49c2-a649-6ef724cc817a">
<img width="524" alt="Screenshot 2024-05-08 at 11 28 21 AM" src="https://github.com/th-ch/youtube-music/assets/424956/bef0adcc-c8f0-4c2d-a639-195b65b57885">

### Expected Behavior

Logging in or approving the app to have access to my google music account. It was signing me in successfully a week or so ago.

### Actual Behavior

I get this error
```
The browser you’re using doesn’t support JavaScript, or has JavaScript turned off.

To keep your Google Account secure, try signing in on a browser that has JavaScript turned on. [Learn more](https://support.google.com/accounts/answer/7675428?hl=en)

If you continue to have problems signing in, please contact your administrator.
```
<img width="536" alt="image" src="https://github.com/th-ch/youtube-music/assets/424956/80be34de-31e0-48f7-9770-2e855e13d68b">

### Enabled plugins

None

### Additional Information

I have noticed some lag opening and navigating the the app for the last couple of weeks.

**Comment by ArjixWasTaken** [comment]
Please override your user agent

![image](https://github.com/th-ch/youtube-music/assets/53124886/26b685ee-671e-4eb0-ac98-d2cb2d8e9eaa)

You can use a site like [this](https://www.whatsmyua.info/) to grab the user agent of your main browser (e.g. firefox, chrome)

**Comment by geekeie** [comment]
Use this Integration Git (Updated and New)
[](https://github.com/geekeie/Youtube-Music-Integration-with-third-party-app/tree/main)

**Comment by PeteKempDojo** [comment]
Overriding user agent has worked for me on v3.3.12 using M2 Mac, thanks @ArjixWasTaken

**Comment by scuestam** [comment]
How do I put the user agent? I'm havings the same issue but when I click in override user agent I can't put anythings.

**Comment by ArjixWasTaken** [comment]
back when I made the suggestion I didn't test it myself, looks like the user agent is hardcoded into the code

**Comment by scuestam** [comment]
I keep getting this when I try to log in.
![imagen](https://github.com/user-attachments/assets/b8e787c0-ad63-44da-aaa7-c83102e447cb)

**Comment by JellyBrick** [comment]
@scuestam see https://github.com/th-ch/youtube-music/issues/2036#issuecomment-2125703592

**Comment by scuestam** [comment]
> @scuestam see [#2036 (comment)](https://github.com/th-ch/youtube-music/issues/2036#issuecomment-2125703592)

Yeah, I click there, but I can't change the user agent, it just get ticked or not. And it makes no difference

**Comment by utccnu** [comment]
> > @scuestam see [#2036 (comment)](https://github.com/th-ch/youtube-music/issues/2036#issuecomment-2125703592)
> 
> Yeah, I click there, it just get ticked or not (I restart the app). It makes no difference

have you figured it out? just installed latest version from brew on M1 pro, same issue. 
Overriding the UA(and restart) didn't fix.
